packages/plugin-compat/extra/fsevents/1.2.11.patch

```diff
@@ -2,20 +2,21 @@ diff --git a/fsevents.js b/fsevents.js
 semver exclusivity ^1
```

Comment:
the API fsevents 1.x exposed makes it quite challenging to watch multiple files so I only watch the path file instead.( same behavior as current patch)